sass/components/_pricing-table.scss

```diff
@@ -0,0 +1,55 @@
+
+.table-section {
+  @include flexbox;
+  width: 100%;
+  padding: 0 5% 3em 5%;
+}
+
+.pricing-table {
+  font-size: .8em;
+  min-width: 16em;
+  margin: 1.5em 1.5em 2em 1.5em;
+  height: 23em;
+  width: 16em;
+}
+
+th {
```

Comment:
Within a component partial, element selectors should be scoped. This is probably meant to apply to `<th>` elements in the pricing table component, but as written, this rule will apply to all `<th>` elements.